### PR TITLE
Changed the parts using strtol to use safe_strtol

### PR DESCRIPTION
**Pull request by minkikim89**

### CODE REVIEW

**Comment by dormando** [comment]
Did you check for test coverage? Anything that could trigger a proper error?

**Comment by dormando** [comment]
(also, thanks! sorry it took so long to check)

**Comment by minkikim89** [comment]
It can be a problem when using the following command.
image

**Comment by dormando** [comment]
If there's a break/fix in the code, do the tests cover them? can you update them if not?

**Comment by minkikim89** [comment]
I added some test codes. However, I haven't found a proper place to add other tests.

**Comment by dormando** [comment]
What were you having trouble finding a place for?

**Comment by minkikim89** [comment]
I already updated some test codes for "flush_all", "slabs_reassign".
But I don't know where I have to add test codes for "verbosity", "slabs_automove".

**Comment by dormando** [comment]
You can just create a new test file for those two right now. "strtol-testing.t". Looks like automove never got its own tests and verbosity isn't really tested anywhere.

**Comment by minkikim89** [comment]
I added tests. Check it please.

**Comment by dormando** [comment]
Looks good. I'll have to check it more carefully when I get back to it. Thanks!

File: memcached.c [hunk 1]
```diff
@@ -5793,11 +5800,11 @@ static void process_flush_all_command(conn *c, token_t *tokens, const size_t nto
...
 
     if (ntokens != (c->noreply ? 3 : 2)) {
-        exptime = strtol(tokens[1].value, NULL, 10);
-        if(errno == ERANGE) {
-            out_string(c, "CLIENT_ERROR bad command line format");
+        if (!safe_strtol(tokens[1].value, &exptime_int)) {
+            out_string(c, "CLIENT_ERROR invalid exptime argument");
             return;
         }
+        exptime = realtime(EXPTIME_TO_POSITIVE_TIME(exptime_int));
```
Comment (dormando) [inline memcached.c:5807]: Note the comment below: realtime() is run after exptime is checked for a 0 value.

Comment (minkikim89) [reply memcached.c]: OK. I fixed it.

**Comment by dormando** [comment]
Released in 1.6.7!

**Pull request closed without merging**